deploy/k8s/README.md

```diff
@@ -0,0 +1,87 @@
+# Deploying on Kubernetes
+This directory contains an example deployment of Synse Server with a containerized
+Emulator Plugin for [Kubernetes][kubernetes].
+
+## Deployments
+As a general note - the Emulator Plugin does not need to be containerized. In
```

Comment:
I would say "containerized separately" since everything in k8s needs to be in a container.